EIPS/eip-7292.md

```diff
@@ -0,0 +1,636 @@
+---
+eip: 7292
```

Comment:
```suggestion
eip: 7291
```
Assigning EIP number using PR number of first opened PR #7291
Please also update the filenames.

Reply:
Thanks for your quick response. I have updated the EIP number to 7291 and changed the filenames.